contrib/syft/template.py

```diff
@@ -0,0 +1,31 @@
+pkgname = "syft"
+pkgver = "1.0.1"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
+    f"-ldflags= -X main.version={pkgver}",
+    "github.com/anchore/syft/cmd/syft",
```

Comment:
```suggestion
    "./cmd/syft",
```

local path seems to work ok